cset-workflow/flow.cylc

```diff
@@ -87,11 +87,20 @@ URL = https://metoffice.github.io/CSET
 
     [[FETCH_DATA]]
 
+    # Dummy tasks needed for workflow scheduling.
     [[process_finish]]
-    # Dummy task needed for workflow scheduling.
     script = true
     platform = localhost
 
+    [[dummy_collate]]
+    inherit = COLLATE
+    script = true
```

Comment:
```suggestion
    script = true
    platform = localhost
```
If you run it on localhost you can skip queueing time, which makes dummy tasks like this a lot faster. This actually runs on the cylc scheduler servers, so it shouldn't be used for tasks doing any work, but for running `true` is perfectly reasonable, given its cheaper than the `slurm` command that would otherwise run.